### ISSUE TITLE
[Bug] "pnpify --sdk" fails if settings.json contains comments

### ISSUE THREAD

**Issue by jgoz** [yarnpkg/berry#521]
**Describe the bug**

Running `yarn pnpify --sdk` fails if `.vscode/settings.json` contains comments.

**To Reproduce**

`.vscode/settings.json`:
```jsonc
{
  // put settings here
}
```

```sh
yarn pnpify --sdk

SyntaxError: Unexpected token / in JSON at position 4
    at JSON.parse (<anonymous>)
    at addVSCodeWorkspaceSettings
```

**Environment if relevant (please complete the following information):**

 - OS: OSX
 - Node version: 10.16.3
 - Yarn version: 2.0.0-rc.6

**Additional context**

VS Code allows comments in its config files. The workaround is to temporarily remove the comments, run `pnpify`, and then restore them, but this might not be obvious to people. Rather than using `JSON.parse` it might be worth using `json5`.

**Comment by arcanis** [comment]
That sounds reasonable 👍

**Comment by arcanis** [comment]
Fixed by #522 ✨
